ci/docker/runtime_functions.sh

```diff
@@ -303,6 +305,7 @@ build_centos7_gpu() {
         -DUSE_DIST_KVSTORE=ON \
         -DBUILD_EXTENSION_PATH=/work/mxnet/example/extensions/lib_external_ops \
         -DUSE_INT64_TENSOR_SIZE=OFF \
+        -DLINK_GFORTRAN=ON \
```

Comment:
Can we have test runtime results for with and without openMP in OpenBLAS for centOS ?